### ISSUE TITLE
Background colors for `base.user.css` not true in Ubuntu

### ISSUE THREAD

**Issue by dajare** [typora/typora-issues#4924]
- [x] Searched existing issues to avoid creating duplicates.

- [x] Confirmed that it can be reproduced in built-in themes without customized css.

   I have checked this with ALL built-in themes (I do not use 3rd party themes).

- [x] Searched http://support.typora.io/

----

**Describe the bug**

In my `base.user.css` file, I have defined `body: {background-color: #f0ead6;}` (= "eggshell"), and with two header levels, `background-color: #ebe3c8;`. However, when "loaded", Typora displays on-screen with these values:

| css | on-screen |
|---|---|
| f0ead6 | f1ead7 |
| ebe3c8 | ece3ca |

This is subtle, but still significant to my eye, and ... ought not to be! The color defined in the CSS should be the color I see on screen. 

**To Reproduce**

1. Define background color in `base.user.css`.
2. Check with "color meter". (I am using [Gpick](http://www.gpick.org/) for this. The difference is visible to the "naked eye", however. I use the same color scheme for LibreOffice Writer, and they stay "true". Comparing Writer and Typora makes the difference very obvious.)
3. See that color does not match the CSS value.

**Expected behavior**

I expect to see the color on screen exactly as defined in the custom CSS file.

**Screenshot**

![screenshot with differences](https://user-images.githubusercontent.com/40625/145790713-afa0f753-31f7-486f-a54f-f941d2b5f814.png)

Yes, subtle (do check with a color meter!), but there shouldn't be any variation.

**Desktop (please complete the following information):**

- OS: Ubuntu 18.04.06 LTS

**Typora Version** = 1.0.3 (registered)

**Comment by abnerlee** [comment]
Did you try launch Typora with flag `--disable-color-correct-rendering`

looks like an upstream issue https://github.com/electron/electron/issues/10732

And see the comment here https://github.com/electron/electron/issues/10732#issuecomment-891802979

> in CSS, hex colors should always be resolved in sRGB, [per spec](https://www.w3.org/TR/css-color-4/#numeric-srgb).

**Comment by dajare** [comment]
> Did you try launch Typora with flag `--disable-color-correct-rendering`

Many thanks for the reply, and the pointers. I have only just now been able to get back to check and test. With the flag as recommended, nothing changed. However, I found [this comment](https://github.com/electron/electron/issues/10732#issuecomment-965670978) in the Electron thread you linked. It included this:

```js
app.commandLine.appendSwitch('disable-color-correct-rendering');
app.commandLine.appendSwitch('force-color-profile', 'srgb');
```
> worked for me for electron v11.0.1

I adapted that to: `$ typora --disable-color-correct-rendering --force-color-profile=srgb`

And that worked in Ubuntu 18.04.06 LTS. Thanks for the help!
